Templates/E-Commerce/Shopify/Shopify to Dynamics 365 - Customers/metadata.json

```diff
@@ -0,0 +1,6 @@
+{
+  "name": "Shopify to Dynamics 365 - Customers",
+  "version": "1.0.0",
+  "tags": [],
+  "additionalSystemsUsed": []
+}
```

Comment:
Add Shopify here in additionalSystemsUsed.

Reply:
Done!